ibex_configs.hjson

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I need a copyright header...

Reply:
I went through a few options for the configuration file, initially I was going to do something super simple you could use awk to pull configuration information from simply, but you quickly end up with complex awk commands or an awkward config file, so I thought just writing some python to do it was a better idea.

I then considered doing a nice readable ASCII art table with the configs, I decided against this for two reasons:

1. Whilst parsing is easy if you keep to strict formatting rules it's brittle
2. As you add more parameters the table ends up getting rather wide and the 'easily-readable' bit is lost so there's not point in doing it in the first place

So I've gone for hjson, the python script checks every config specifies the same configuration options. I considered having a default section then specific overrides for different configs but thought it was better to able to look at a single config and immediately know what parameters are getting set to what values.